### PR TITLE
code refactor

### PR DESCRIPTION
**Pull request by MalwinaKozak**
delete not used functions,
move GPS class to /sensors,
change imports

### CODE REVIEW

File: code/sensors/GPS.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+            self.filter_gps(lat, lon, False)
+
+            frame_data['latitude'] = self.f[0].x[0][0]
+            frame_data['longitude'] = self.f[1].x[0][0]
+
+    def callback_additional(self, frame_data):
+        speed = frame_data['vspd']
+        delta_distance = speed * 0.1 / 3600
+        self.kilometers_driven += delta_distance
```
Comment (cepekLP) [inline code/sensors/GPS.py:37]: we calculate kilometers driven in both callbacks

File: code/sensors/GPS.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            self.lon_current = self.f[1].x[0][0]
+
+    def calc_kilometers_driven(self):
+        if self.lat_prev is None or self.lon_prev is None:
+            self.lat_prev = self.lat_current
+            self.lon_prev = self.lon_current
+        delta_kilometers = self.distance(self.lat_prev, self.lon_prev, self.lat_current, self.lon_current)
+        self.kilometers_driven += delta_kilometers
+
+    def distance(self, lat1, lon1, lat2, lon2):
```
Comment (cepekLP) [inline code/sensors/GPS.py:79]: use values from the class -> self.lat1 ....

File: code/sensors/GPS.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        self.lon_prev = self.f[1].x[0][0]
+        for i in range(2):
+            self.f[i].predict()
+        if not signal_loss:  
+            self.f[0].update(lat)
+            self.f[1].update(lon)
+            self.lat_current = self.f[0].x[0][0]
+            self.lon_current = self.f[1].x[0][0]
+
+    def calc_kilometers_driven(self):
```
Comment (cepekLP) [inline code/sensors/GPS.py:72]: unused function

**Review by cepekLP — CHANGES_REQUESTED** [review]

File: code/sensors/GPS.py (hunk 2 above)
Comment (MalwinaKozak) [reply code/sensors/GPS.py:79]: function deleted

**Review by cepekLP — APPROVED** [review]
👍

**Pull request merged**